pkgs/top-level/aliases.nix

```diff
@@ -377,6 +377,7 @@ mapAliases ({
   fast-neural-doodle = throw "fast-neural-doodle has been removed, as the upstream project has been abandoned"; # Added 2020-03-28
   fastnlo = fastnlo_toolkit; # Added 2021-04-24
   fedora-coreos-config-transpiler = throw "fedora-coreos-config-transpiler has been renamed to 'butane'"; # Added 2021-04-13
+  feedreader = throw "FeedReader is no longer activily maintained since 2019. The developer is working on a spiritual successor called NewsFlash."; # Added 2022-05-03
```

Comment:
One small nit, not dot at the end, I think

Reply:
I think thats whatever. There is no general format if you browse over all the entries. A must is the package name at the beginning otherwise you can't find it.

Reply:
```suggestion
  feedreader = throw "feedreader is no longer activily maintained since 2019. The developer is working on a spiritual successor called NewsFlash."; # Added 2022-05-03
```

Reply:
For some reason GitHub added @Mindavi as Co-Author and not @SuperSandro2000 who suggested the change. But i think that's not very important.